website_and_docs/content/documentation/support_packages/chrome_devtools.en.md

```diff
@@ -5,16 +5,21 @@ weight: 5
 aliases: ["/documentation/en/support_packages/chrome_devtools/"]
 ---
 
-Selenium 4 alpha versions have much awaited native support for Chrome DevTools 
-Protocol through "DevTools" interface. This helps us getting Chrome Development 
-properties such as Application Cache, Fetch, Network, Performance, Profiler, 
-Resource Timing, Security and Target CDP domains etc.
-
-Chrome DevTools is a set of web developer tools built directly into the Google 
-Chrome browser. DevTools can help you edit pages on-the-fly and diagnose 
-problems quickly, which ultimately helps you build better websites, faster.
-
-## Emulate Geo Location:
+{{% pageinfo color="warning" %}}
+<p class="lead">
+  While Selenium 4 provides direct access to Chrome DevTools Protocol, it is highly encouraged that
+you use the <a href="../../webdriver/bidi_apis">WebDriver Bidi APIs</a> instead.
+</p>
+{{% /pageinfo %}}
+
+Chrome DevTools is a set of web developer tools built directly into the Google
```

Comment:
I'm wary of confusing people between "devtools" (available on many browsers, including Safari, Firefox, and IE) and "CDP" (a protocol that the chromium-based browsers, and to an extent Firefox, implements) It turns out that Chromium's DevTools are based on CDP, but that's not the case for all browsers.

Perhaps spell that out: "Chrome's DevTools are a set of web developer tools built directly into Chromium-based browsers. They use the CDP to communicate with the browser...."

Reply:
Excellent point. I'm also going to rename this page to "Chrome DevTools Protocol." Not sure it is worth updating the file name as well.

Reply:
How about something like?

> Many browsers provide "DevTools" -- a set of tools that are integrated with the browser that developers can use to debug web apps and explore the performance of their pages. Google Chrome's DevTools make use of a protocol called the Chrome DevTools Protocol (or "CDP" for short). As the name suggests, this is not designed for testing, nor to have a stable API, so functionality is highly dependent on the version of the browser.
>
> WebDriver Bidi is the next generation of the W3C [WebDriver protocol](https://w3c.github.io/webdriver-bidi/) and aims to provide a stable API implemented by all browsers, but it's not yet complete. Until it is, Selenium provides access to the CDP for those browsers that implement it (such as Google Chrome, or Microsoft Edge, and Firefox), allowing you to enhance your tests in interesting ways. Some examples of what you can do with it are given below.
```